docs/challenges.rst

```diff
@@ -1,85 +1,82 @@
-* Offering a specified temporary certificate on a web site (the TLS-SNI-01 challenge) 
-* Posting a specified DNS record in the domain name system (the DNS-01 challenge)
+* Posting a specified file in a specified location on a web site (the HTTP-01_ challenge)
+* Offering a specified temporary certificate on a web site (the TLS-SNI-01_ challenge) 
+* Posting a specified DNS record in the domain name system (the DNS-01_ challenge)
+
+The DNS-01 challenge, in particular, requires configuration of a DNS server on
+port 53, though that's often not the same machine as your webserver. 
 
-It’s possible to complete each type of challenge *automatically* (Certbot directly makes the necessary 
-changes itself, or runs another program that does so), or *manually* (Certbot tells you to make a 
-certain change, and you edit a configuration file of some kind in order to accomplish it). Certbot's 
+It’s possible to complete each type of challenge *automatically* (Certbot directly makes the necessary 
+changes itself, or runs another program that does so), or *manually* (Certbot tells you to make a 
+certain change, and you edit a configuration file of some kind in order to accomplish it). Certbot's 
 design favors performing challenges automatically, and this is the normal case for most users of Certbot.
 
-Some plugins offer an *authenticator*, meaning that they can satisfy challenges:
-
-* Apache plugin: (TLS-SNI-01) Tries to edit your Apache configuration files to temporarily serve 
-  a Certbot-generated certificate for a specified name. Use the Apache plugin when you're running 
-  Certbot on a web server with Apache listening on port 443.
-* NGINX plugin: (TLS-SNI-01) Tries to edit your NGINX configuration files to temporarily serve a 
-  Certbot-generated certificate for a specified name. Use the NGINX plugin when you're running 
-  Certbot on a web server with NGINX listening on port 443.
-* Webroot plugin: (HTTP-01) Tries to place a file where it can be served over HTTP on port 80 by a
-  web server running on your system. Use the Webroot plugin when you're running Certbot on 
-  a web server with any server application listening on port 80 serving files from a folder on disk in response.
-* Standalone plugin: (TLS-SNI-01 or HTTP-01) Tries to run a temporary web server listening on either HTTP on 
-  port 80 (for HTTP-01) or HTTPS on port 443 (for TLS-SNI-01). Use the Standalone plugin if no existing program 
-  is listening to these ports. Choose TLS-SNI-01 or HTTP-01 using the `--preferred-challenges` option.
-* Manual plugin: (DNS-01 or HTTP-01) Either tells you what changes to make to your configuration or updates 
-  your DNS records using an external script (for DNS-01) or your webroot (for HTTP-01). Use the Manual 
-  plugin if you have the technical knowledge to make configuration changes yourself when asked to do so. 
+.. _challenges: https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-7
```

Comment:
If you want these to be links rather than section headers you want to have 
NAME_
not
.. _NAME